src/features/fwiCalculator/components/CalculationDocumentation.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Card, CardContent, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles({
+  root: {
+    marginTop: 15,
+    marginBottom: 15,
+    width: 600
+  },
+  title: {
+    fontSize: 18,
+    marginBottom: 10,
+    color: '#000000'
+  },
+  bodyText: {
+    color: '#333333',
+    fontSize: 14,
+    marginBottom: 10
+  }
+})
+
+export const CalculationDocumentation = () => {
+  const classes = useStyles()
+
+  return (
+    <Card
+      className={classes.root}
+      id="documentation-card"
+      data-testid="documentation-card"
```

Comment:
Can you write a test using this test id? test if this component appears when the result is fetched. Also, can you make the id more specific to percentile? documentation-card is too generic